plugins/org.jkiss.dbeaver.core/OSGI-INF/l10n/bundle_zh.properties

```diff
@@ -147,3 +148,41 @@ wizard.org.jkiss.dbeaver.core.wizard.import.scripts.description = SQL \u811A\u67
 wizard.org.jkiss.dbeaver.core.wizard.import.scripts.name        = \u811A\u672C
 wizard.org.jkiss.dbeaver.core.wizard.new.connection.description = \u6570\u636E\u5E93\u8FDE\u63A5
 wizard.org.jkiss.dbeaver.core.wizard.new.connection.name        = \u6570\u636E\u5E93\u8FDE\u63A5
+extension-point.org.jkiss.dbeaver.clearHistoryHandler.name = \u6E05\u9664\u5386\u53F2\u7BA1\u7406\u5668
+command.org.jkiss.dbeaver.core.commit.label = \u63D0\u4EA4
```

Comment:
Unused localization
```suggestion
```

Reply:
Do I need to delete the unused parts of the localization files for other languages?